### ISSUE TITLE
The default value of patience in EarlyStopping's doc is inconsistent with the value in parameters

### ISSUE THREAD

**Issue by NickYi1990** [PyTorchLightning/pytorch-lightning#1860]
## 📚 Documentation

The default value of patience in EarlyStopping's doc is inconsistent with the value in parameters

```
class EarlyStopping(Callback):
    r"""
    Args:
        monitor: quantity to be monitored. Default: ``'val_loss'``.
        min_delta: minimum change in the monitored quantity
            to qualify as an improvement, i.e. an absolute
            change of less than `min_delta`, will count as no
            improvement. Default: ``0``.
        patience: number of epochs with no improvement
            after which training will be stopped. Default: ``0``.
        verbose: verbosity mode. Default: ``False``.
        mode: one of {auto, min, max}. In `min` mode,
            training will stop when the quantity
            monitored has stopped decreasing; in `max`
            mode it will stop when the quantity
            monitored has stopped increasing; in `auto`
            mode, the direction is automatically inferred
            from the name of the monitored quantity. Default: ``'auto'``.
        strict: whether to crash the training if `monitor` is
            not found in the metrics. Default: ``True``.
    Example::
        >>> from pytorch_lightning import Trainer
        >>> from pytorch_lightning.callbacks import EarlyStopping
        >>> early_stopping = EarlyStopping('val_loss')
        >>> trainer = Trainer(early_stop_callback=early_stopping)
    """
    mode_dict = {
        'min': torch.lt,
        'max': torch.gt,
    }

    def __init__(self, monitor: str = 'val_loss', min_delta: float = 0.0, patience: int = 3,
                 verbose: bool = False, mode: str = 'auto', strict: bool = True):
```

**Comment by williamFalcon** [comment]
@NickYi1990 good catch. Submit a PR?

**Comment by ydcjeff** [comment]
@Borda this has been fixed on latest docs, but not on `0.8.5` docs

**Comment by Borda** [comment]
@ydcjeff we most likely won't fix pas docs, sorry
